locale/zh_CN.po

```diff
@@ -1050,11 +1050,11 @@ msgstr "无法生成调试报告。"
 #, c-format
 msgid ""
 "Processing debug report has failed, leaving the files in \"%s\" directory."
-msgstr "处理调试报告失败, 文件被保存在目录 \"%s\" 中。"
+msgstr "处理调试报告失败, 文件被保存在 \"%s\" 目录。"
 
 #: ../src/common/debugrpt.cpp:574
 msgid "A debug report has been generated. It can be found in"
-msgstr "产生了一份调试报告。 位于"
+msgstr "产生了一份调试报告，位于"
```

Comment:
Same here. The original text is a period `.` rather than a comma `,`.

Reply:
replaced with ，so the context connects. This is human translation, not MT.